pkg/objectfile/cache.go

```diff
@@ -32,11 +32,11 @@ type cache struct {
 var ErrNoFile = errors.New("cannot load object file for mappings with empty file")
 
 // NewCache creates a new cache for object files.
-func NewCache(size int) *cache {
+func NewCache(size int, profiligDuration time.Duration) *cache {
 	return &cache{
 		cache: burrow.New(
 			burrow.WithMaximumSize(size),
-			burrow.WithExpireAfterAccess(time.Minute), // Deprecate it if it is not used for 6 profiling cycles (default).
+			burrow.WithExpireAfterAccess(6*profiligDuration), // Deprecate it if it is not used for 6 profiling cycles (default).
```

Comment:
With the default value (10s), it rounds up to one minute. Other than that, it doesn't have any specific meaning. I think the timed caches should be relative to profiling duration otherwise if it changes, behavior is unpredictable to me.